frontend/src/component/Dashboard/UserStatus.styl

```diff
@@ -3,12 +3,12 @@
 .userstatus
   width 27%
   position relative
-  top -60px
+  top 120px
   border-radius 5px
   padding 40px
-  background-color lightGrey
+  background-color offWhite
   text-align center
-  font-size 23px
+  font-size 20px
```

Comment:
Would it make sense to name this font size? (in Variables.styl)

Reply:
Updated